File: core/src/com/spacefist/components/NullInputComponent.java
package com.spacefist.components;

import com.spacefist.GameData;
import com.spacefist.components.abst.InputComponent;
import com.spacefist.entities.Entity;

/**
 * For use by entities that do not use input.
 */
public class NullInputComponent implements InputComponent {
    public void update(GameData gameData, Entity obj) { }
}

File: core/src/com/spacefist/components/NullInputComponent.kt
package com.spacefist.components

import com.spacefist.GameData
import com.spacefist.components.abst.InputComponent
import com.spacefist.entities.Entity

/**
 * For use by entities that do not use input.
 */
class NullInputComponent : InputComponent {
    override fun update(gameData: GameData, obj: Entity) {}
}
